FeatureFileGenerator/settings.py

```diff
@@ -0,0 +1,3 @@
+#target process stuff
+tp_uri = "https://bluefruit.tpondemand.com/api/v1/"
+token = "NTE6V0FMZWJ1c2dkTkZ1aDVKdzRSM0FHSGFYM3pJd0IraUhnR016UzRqS0NiUT0="
```

Comment:
i can imagine having an api key hardcoded into source in a public repo probably isnt a good idea, but im not quite sure where else it can go

Reply:
ah yeah, whoops, lets remove that file and re do things :D

Reply:
Ive deleted the token from having access to TP, will push changes in a sec :D Would have been bad for everyone ever to have access to our date... :D